tests/test-sessions/test-sessions-common/src/main/java/org/eclipse/jetty/server/session/TestHttpChannelCompleteListener.java

```diff
@@ -22,15 +22,15 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import org.eclipse.jetty.server.HttpChannel.Listener;
-import org.eclipse.jetty.server.HttpChannelState;
 import org.eclipse.jetty.server.Request;
-import org.eclipse.jetty.server.handler.ContextHandler.Context;
-import org.eclipse.jetty.server.handler.ContextHandler.ContextScopeListener;
 
-public class TestContextScopeListener implements ContextScopeListener
+public class TestHttpChannelCompleteListener implements Listener
 {
     AtomicReference<CountDownLatch> _exitSynchronizer = new AtomicReference<>();
-    boolean listenerAdded = false;
+
+    public TestHttpChannelCompleteListener()
+    {
+    }
```

Comment:
The constructor is not necessary, remove.

Also, `getExitSynchronizer()` is never used, is that intended?